.circleci/local_publish_helpers.sh

```diff
@@ -71,7 +71,22 @@ function generatePkgCli {
   # Build pkg cli
   cp package.json ../build/node_modules/package.json
   if [[ "$CIRCLE_BRANCH" == "release" ]] || [[ "$CIRCLE_BRANCH" =~ ^run-e2e-with-rc\/.* ]] || [[ "$CIRCLE_BRANCH" =~ ^release_rc\/.* ]] || [[ "$CIRCLE_BRANCH" =~ ^tagged-release ]]; then
-    npx pkg -t node14-macos-x64,node14-linux-x64,node14-linux-arm64,node14-win-x64 ../build/node_modules --out-path ../out
+    # This will generate a file 'amplify-pkg' in the '../tmp-out' directory for our arm64 binary.
+    # The file name does not include the generated <platform>-<arch> suffixes because there is a single target.
+    npx pkg --no-bytecode -t node14-linux-arm64 ../build/node_modules --out-path ../tmp-out
```

Comment:
Could you try `-o` aka `--output` parameter ti give it proper name and avoid extra tmp dir ?

see https://www.npmjs.com/package/pkg .